python/flexflow/core/__init__.py

```diff
@@ -33,11 +33,20 @@
     from .flexflow_cffi import *
 
   # check if use native python interpreter
-  if flexflow_python_interpreter() == 'native':  
-    from .flexflow_pybind11_internal import begin_flexflow_task, finish_flexflow_task
+  if flexflow_python_interpreter() == 'native': 
     print("Using native python")
-    begin_flexflow_task(sys.argv)
-    atexit.register(finish_flexflow_task)
+    if flexflow_python_binding() == 'pybind11':
+      from .flexflow_pybind11_internal import begin_flexflow_task, finish_flexflow_task
```

Comment:
Aren't we getting rid of pybind support?

Reply:
Yes, but that will be in another PR, so I will just keep the pybind in this one.